### PR TITLE
2491 Get Support Self-service

### PR DESCRIPTION
**Pull request by markhallen**
### Context

The support team receive many requests that can be handled by directing the requestor to guidance pages already on the app.

### Changes proposed in this pull request

Improving the support form to suggest helpful links when the user answers questions along the journey. This is the first iteration and only touches the first step on the get support journey.

### Guidance to review

Options and suggestions are loaded from `config/support_tickets.yml`

Check the options are displayed when selecting "Which of these best describes you?" on /get-support/describe-yourself

image

image

### CODE REVIEW

**Comment by markhallen** [comment]
[Iterate contact form to increase self-service](https://trello.com/c/QMNEPt1A/2491-iterate-contact-form-to-increase-self-service)

File: config/support_tickets.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  - title: Resetting 4G routers
+    resource: "/managing_your_4g_wireless_routers"
+  - title: Finding out about internet access needs
+    resource: "/finding-out-about-internet-access-needs"
+- value: :digital_education_platforms
+  label: Digital education platforms (Google Workspace for Education Fundamentals or Microsoft 365 Education)
+- value: :technology_training_and_support
+  label: Technology training and support for schools and colleges
+- value: :something_else
+  label: Something else
```
Comment (ltello) [inline config/support_tickets.yml:60]: Nice having all these in a config file!

**Review by ltello — APPROVED** [review]
Great

File: config/support_tickets.yml (hunk 1 above)
Comment (markhallen) [reply config/support_tickets.yml:60]: Thanks. It really didn't feel right in the class itself.

**Pull request merged**